py2pack/__init__.py

```diff
@@ -56,6 +65,32 @@ def pypi_json(project, release=None):
     return pypimeta
 
 
+def pypi_text_file(pkg_info_path):
+    with open(pkg_info_path, 'r') as pkg_info_file:
+        pkg_info_lines = parser.Parser().parse(pkg_info_file)
+    pkg_info_dict = {}
+    for key, value in pkg_info_lines.items():
+        key = key.lower().replace('-', '_')
+        if key in {'classifiers', 'requires_dist', 'provides_extra'}:
+            val = pkg_info_dict.get(key)
+            if val is None:
+                val = []
+                pkg_info_dict[key] = val
+            val.append(value)
+        else:
+            pkg_info_dict[key] = value
+    return {'info': pkg_info_dict, 'urls': []}
+
+
+def pypi_json_file(file_path):
+    js = json.load(open(file_path))
```

Comment:
maybe it's better to use the file context manager here:

```suggestion
    with open(file_path, 'r') as json_file:
        js = json.load(json_file)
```